CHANGELOG.md

```diff
@@ -78,6 +78,7 @@ Check the history of the branch FIXME.
 * [6179](https://github.com/grafana/loki/pull/6179) **chaudum**: Add new HTTP endpoint to delete ingester ring token file and shutdown process gracefully
 * [5997](https://github.com/grafana/loki/pull/5997) **simonswine**: Querier: parallize label queries to both stores.
 * [5406](https://github.com/grafana/loki/pull/5406) **ctovena**: Revise the configuration parameters that configure the usage report to grafana.com.
+* [7597](https://github.com/grafana/loki/pull/7597) **redbaron**: allow ratelimiting by label
```

Comment:
Hey @redbaron . sorry, for the delay. was on sick leave. 
Thanks for addressing all the comments. I am happy to merge this PR. 
However, the last thing that I ask you to fix is to move the changelog entry that you added. now, it's placed in V2.7.0  changelogs ... could you please move it to the next line after https://github.com/grafana/loki/pull/7597/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR36

Reply:
maybe it was moved when you pulled the latest main.... sorry for so many cycles of the review ;)

Reply:
nevermind, I am fixing it